internal/pkg/istio/config_factory.go

```diff
@@ -160,8 +161,11 @@ func (cf *ConfigFactory) IngressGateway() (*v1alpha3.Gateway, error) {
 	return gateway, nil
 }
 
+// EnvoyFilters returns patches for SNI filters matching SNIs of exported services in federation ingress gateway.
+// These patches add SNI compatible with https://datatracker.ietf.org/doc/html/rfc952 required by OpenShift Router.
+// This function returns empty list when the local ingress type is "istio".
```

Comment:
Nit: it doesn't return empty list (or slice), but `nil` pointer. Either change godoc or implementation :)

Reply:
Thanks, I fixed this comment.